core/utils/create_child_instance_from_parent.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+
+# This file is part of memopol.
+#
+# memopol is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of
+# the License, or any later version.
+#
+# memopol is distributed in the hope that it will
+# be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU General Affero Public
+# License along with django-representatives.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright (C) 2015 Arnaud Fabre <af@laquadrature.net>
+from __future__ import absolute_import
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
```

Comment:
Seems like this is used when a Representative save signal is emited to create a MemopolRepresentative in the database. It's been made generic but it seems used only there.

Maybe it would have been better to use django-swappable-models instead to replace the Representative model provided by django-representatives.